config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -57,6 +57,20 @@ presets:
       items:
       - key: bootstrapper-key
         path: bootstrapper-key
+- labels:
+    preset-cluster-api-provider-vsphere-gcs-creds: "true"
+  volumes:
```

Comment:
Hi @codenrhoden,

I recommend adding the following sibling of `labels` and `volumes` between the two:

```yaml
- labels:
    preset-cluster-api-provider-vsphere-gcs-creds: "true"
  env:
  - name: GCR_KEY_FILE
    value: /root/.capv/keyfile.json
  volumes:
```

This way you don't have to use `-k /root/.capv/keyfile.json` in any of the job defs below since `GCR_KEY_FILE` in `hack/release.sh` [defaults](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/127436f4c979bf81404ca5d100c148a83a5b0d2a/hack/release.sh#L46) to the value of the env var if the flag isn' set.